src/AddTodoForm.js

```diff
@@ -1,11 +1,21 @@
 import React from 'react';
 
-function AddTodoForm() {
+const AddTodoForm = (props) => {
+
+    const handleAddTodo = (event) => {
+        event.preventDefault();
+        // setNewTodo(event.target.value);
+        var todoTitle = event.target.title.value;
+        console.log(todoTitle);
+        props.onAddTodo(todoTitle);
```

Comment:
@olegkolyaka78 I saw your comment on the instructions.  This looks valid how you handled this.  I'll drop a note in slack to the course owner and verify the instructions.